src/PluginsManager/Microsoft.Performance.Toolkit.PluginsManager.Core/Manager/IPluginsManager.cs

```diff
@@ -80,5 +85,146 @@ public interface IPluginsManager
         ///     The directory to load resource assemblies from.
         /// </param>
         void LoadAdditionalPluginResources(string directory);
+
+        /// <summary>
+        ///     Gets all installed plugins
+        /// </summary>
+        /// <param name="cancellationToken">
+        ///     Signals that the caller wishes to cancel the operation.
+        /// </param>
+        /// <returns>
+        ///     A collection of installed plugins.
+        /// </returns>
+        Task<IReadOnlyCollection<InstalledPlugin>> GetInstalledPlugins(CancellationToken cancellationToken);
+
+        /// <summary>
+        ///      Checks if any plugin with the given ID has been installed to the plugin registry.
+        /// </summary>
+        /// <param name="pluginId">
+        ///     A plugin identifier.
+        /// </param>
+        /// <param name="cancellationToken">
+        ///     Signals that the caller wishes to cancel the operation.
+        /// </param>
+        /// <returns>
+        ///     <c>true</c> if the plugin is currently installed. <c>false</c> otherwise.
+        /// </returns>
+        Task<bool> IsPluginInstalled(string pluginId, CancellationToken cancellationToken);
```

Comment:
1. Why would someone need to call this?
2. What value should `pluginId` be? Is it different from the `PluginIdentity` class?